inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -85,16 +91,112 @@ class XmlSerializer : public ngraph::AttributeVisitor {
 public:
     XmlSerializer(pugi::xml_node& data,
                   std::ostream& bin_data,
-                  std::string& node_type_name)
+                  std::string& node_type_name,
+                  const std::map<std::string, ngraph::OpSet>& custom_opsets)
         : m_xml_node(data)
         , m_bin_data(bin_data)
-        , m_node_type_name(node_type_name) {
+        , m_node_type_name(node_type_name)
+        , m_custom_opsets(custom_opsets) {
+    }
+
+    std::vector<std::string> map_type_from_body(const pugi::xml_node& xml_node,
+        const std::string& map_type) {
+        std::vector<std::string> output;
+        if (map_type == "Result" || map_type == "Parameter") {
+            for (pugi::xml_node node : xml_node.child("body").child("layers")) {
+                auto param = map_type.compare(node.attribute("type").value());
+                if (!param) {
```

Comment:
nitpick: Looks like we can do a compare in-place.

Reply:
Corrected.